Functions/xASL_adm_RemoveDirectories.m

```diff
@@ -22,9 +22,9 @@ function xASL_adm_RemoveDirectories(root)
     
     % ExploreASL
     try
-        warning('off', 'all');
+        warning('off', 'MATLAB:rmpath:DirNotFound');
```

Comment:
great, perhaps add some comments about what we do here, and why we use the try-catch construction for this. It seems not required? Matlab never gives an error with `rmpath`, only warnings? I would avoid try-catch constructions if we can before the code becomes too convoluted

Reply:
Yeah, but you get like a hundred warnings and I just wanted to make sure that the pipeline is not killed because of this script which is only there to improve test independence.

Reply:
I am a bit confused:)
It is OK to switch off warnings for rmpath - makes total sense to avoid too much unnecessary warnings - so just add a comment explaining in the code why you do this.

But the question was mostly the try-catch - why is that needed. Can theoretically `warning` crash or `rmpath` crash - opinion of Henk is that these cannot really crash, or can they?

Reply:
> Can theoretically warning crash or rmpath crash - opinion of Henk is that these cannot really crash, or can they?

I actually wasn't sure about that, but I thought better safe than sorry. Didn't knew you had so strong opinions on this :D

43f5305 👍